scripts/deploy.sh

```diff
@@ -56,12 +56,13 @@ if [[ "${DEPLOY_AIO}" == "yes" ]]; then
   export DEPLOY_HAPROXY="yes"
   if [[ ! -d /etc/openstack_deploy/ ]]; then
     ./scripts/bootstrap-aio.sh
+    # move OSA variables file to AIO location.
```

Comment:
That will be part of the official documentation, leaving it in a comment is the wrong place for docs.